<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>olymain</class>
 <widget class="QMainWindow" name="olymain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1373</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Olympus</string>
  </property>
  <property name="styleSheet">
   <string notr="true">button:hover{
	background-color=yellow;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="controls_container">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>691</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>50</horstretch>
      <verstretch>50</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Sala de control</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>631</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="control_layout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>631</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="options_layout"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="robots_container">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>701</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color = 
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255));</string>
    </property>
    <property name="title">
     <string>Robots</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>681</width>
       <height>651</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="robots_layout"/>
    </widget>
   </widget>
   <zorder>controls_container</zorder>
   <zorder>robots_container</zorder>
   <zorder>horizontalLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1373</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAcerca">
    <property name="title">
     <string>Acerca</string>
    </property>
    <addaction name="actionAcerca_de_Olympus"/>
   </widget>
   <widget class="QMenu" name="menuOpciones_2">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionPreferencias"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuOpciones_2"/>
   <addaction name="menuAcerca"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAcerca_de_Olympus">
   <property name="text">
    <string>Acerca de Olympus</string>
   </property>
  </action>
  <action name="actionPreferencias">
   <property name="text">
    <string>Preferencias</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
